packages/feathernotes/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/tsujan/FeatherNotes
+TERMUX_PKG_DESCRIPTION="Lightweight Qt Notes-Manager for Linux"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="Yisus7u7"
+TERMUX_PKG_VERSION=0.9.0
+TERMUX_PKG_SRCURL=https://github.com/Yisus7u7/shpkg-builds/releases/download/${TERMUX_PKG_VERSION}/feathernotes-${TERMUX_PKG_VERSION}.tar.xz
```

Comment:
Custom sources aren't allowed. If you have made any changes to the source code, bundle patches within directory where build.sh is stored and set TERMUX_PKG_SRCURL to the original upstream.